datafusion/common/src/config.rs

```diff
@@ -282,7 +282,7 @@ config_namespace! {
         /// Should DataFusion support recursive CTEs
         /// Defaults to false since this feature is a work in progress and may not
         /// behave as expected
-        pub enable_recursive_ctes: bool, default = false
+        pub enable_recursive_ctes: bool, default = true
```

Comment:
to set back to false. Need to provide a method to set this at the root config level